src/cl/ec.cl

```diff
@@ -0,0 +1,123 @@
+// Elliptic curve operations (Short Weierstrass Jacobian form)
+
+#define POINT_ZERO ((POINT_projective){FIELD_ZERO, FIELD_ONE, FIELD_ZERO})
+
+typedef struct {
+  FIELD x;
+  FIELD y;
+  #if FIELD_LIMB_BITS == 32
+    uint _padding;
+  #endif
```

Comment:
This should be removed. That's only needed for `pairing` and not for `blst`. See https://github.com/filecoin-project/bellperson/pull/211 for more information.

Reply:
done